services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptDatasets/individualDataset/activatedLLMPromptsSection.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from "react";
+import { useQuery, } from 'react-query';
+import moment from 'moment';
+
+import ViewLLMPromptModal from '../shared/viewLLMPromptModal'
+import { DataTable, Spinner, } from '../../../../../Shared/index';
+import { percentAccuracy, } from '../../../../helpers/evidence/miscHelpers';
+import { fetchActivatedLLMPromptsForPrompt } from '../../../../utils/evidence/genAIAPIs';
+import { ActivatedLLMPromptInterface, } from '../../../../interfaces/evidenceInterfaces';
+
+const ActivatedLLMPromptsSection = ({ promptId, datasetId, individualDatasetLinkBase, activeActivatedLLMPrompt, }: { promptId: number, datasetId: number, individualDatasetLinkBase: string, activeActivatedLLMPrompt: ActivatedLLMPromptInterface }) => {
+  const [activatedLLMPrompts, setActivatedLLMPrompts] = React.useState<any>(null)
+  const [llmPromptIdForModal, setLLMPromptIdForModal] = React.useState<number>(null)
+  const [showViewPromptModal, setShowViewPromptModal] = React.useState<boolean>(false);
+  const [loading, setLoading] = React.useState<boolean>(true)
+
+  const { data: activatedLLMPromptsData } = useQuery({
+    queryKey: [`activated-llm-prompts-for-${promptId}`, promptId],
+    queryFn: fetchActivatedLLMPromptsForPrompt,
+  });
+
+  React.useEffect(() => {
+    if (activatedLLMPromptsData?.data) {
+      const { activated_llm_prompts, } = activatedLLMPromptsData.data;
+      setActivatedLLMPrompts(activated_llm_prompts);
+      setLoading(false);
+    }
+  }, [activatedLLMPromptsData]);
+
+  function openViewPromptModal(llmPromptId) {
+    setLLMPromptIdForModal(llmPromptId)
+    setShowViewPromptModal(true)
+  }
+
+  function closeViewPromptModal() {
+    setShowViewPromptModal(false)
+    setLLMPromptIdForModal(null)
+  }
+
+  const dataTableFields = [
+    {
+      name: 'Trial Number',
+      attribute: 'trialNumber',
+      width: '60px',
+      headerClassName: 'center-content',
+      rowSectionClassName: 'center-content',
+      noTooltip: true
+    },
+    {
+      name: 'Dataset',
+      attribute: 'dataset',
+      width: '60px',
+      headerClassName: 'center-content',
+      rowSectionClassName: 'center-content',
+      noTooltip: true
+    },
+    {
+      name: 'Activated',
+      attribute: 'created',
+      width: '55px',
+      rowSectionClassName: 'allow-wrap',
+      noTooltip: true
+    },
+    {
+      name: 'Prompt Template',
+      attribute: 'llmPromptName',
+      width: '72px',
+      headerClassName: 'center-content',
+      rowSectionClassName: 'center-content allow-wrap',
```

Comment:
Just an idea... I think at some point down the road, we should probably update Backpack to include these different class instances so we have them all documented somewhere